config/database.yml

```diff
@@ -11,3 +11,7 @@ test:
   adapter: postgresql
   database: travis_ci_test
   username: postgres
+
+production:
+  <<: *default
+  database: gi_bill_comparison_tool_production
```

Comment:
@plusjeff Does the change to this file do anything for using a relative URL?  Or is it unrelated and just in here anyways?